packages/contract-addresses/src/index.ts

```diff
@@ -42,7 +44,8 @@ const networkToAddresses: { [networkId: number]: ContractAddresses } = {
         orderValidator: NULL_ADDRESS,
         zrxToken: '0xe41d2489571d322189246dafa5ebde1f4699f498',
         etherToken: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
-        assetProxyOwner: NULL_ADDRESS,
+        assetProxyOwner: '0xdffe798c7172dd6deb32baee68af322e8f495ce0',
```

Comment:
Nit: Will this be removed after the full migration?

Reply:
Yep, this is still needed for the mainnet migrations.